setup.py

```diff
@@ -144,7 +166,7 @@ def check_for_openmp():
 EXTRA_COMPILE_ARGS = ['-O3', '-std=c++11', '-pedantic']
 EXTRA_LINK_ARGS = []
 
-if sys.platform == 'darwin':
+if sys.platform == 'darwin' and 'clang' in os.getenv('CC', 'cc'):
```

Comment:
Ah yes... I was trying to get gcc compilation working on OSX, because the gcc compiled version is much faster than the clang one, for reasons that aren't clear. This is probably what's borking it on @ctb 's machine as well.